### PR TITLE
Feature/v2

### PR DESCRIPTION
**Pull request by corwinc**
Includes:
- How it Works section
- Jurisdictions (map) section

### CODE REVIEW

File: src/containers/jurisdictions/JurisdictionLabel.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_SM}px) {
+    flex-basis: 10%;
+    font-size: 14px;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
+    flex-basis: 20%;
+  }
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_LG}px) {
```
Comment (UnsungHero97) [inline src/containers/jurisdictions/JurisdictionLabel.js]: Check your logic here. `props.x` and `props.y` are undefined.

screen shot 2018-06-25 at 10 00 04 am

File: src/containers/howitworks/Content01.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import Image01 from '../../assets/images/01_problem_definition.png';
```
Comment (UnsungHero97) [inline src/containers/howitworks/Content01.js]: Please use a more descriptive name.

File: src/containers/howitworks/Content02.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import Image02 from '../../assets/images/02_data_sharing_agreements.png';
```
Comment (UnsungHero97) [inline src/containers/howitworks/Content02.js]: Please use a more descriptive name.

File: src/containers/howitworks/Content03.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Image03 from '../../assets/images/03_establishing_data_access.png';
```
Comment (UnsungHero97) [inline src/containers/howitworks/Content03.js]: Please use a more descriptive name.

File: src/containers/howitworks/Content04.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import Image04 from '../../assets/images/04_specific_services.png';
```
Comment (UnsungHero97) [inline src/containers/howitworks/Content04.js]: Please use a more descriptive name.

File: src/containers/howitworks/HowItWorksSection.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+const StyledCroppedSubtitle = StyledSectionSubTitle.extend`
+  margin-bottom: 60px;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_LG}px) {
+    width: 46%;
+  }
+`;
+
+const HowItWorksSection = () => (
```
Comment (UnsungHero97) [inline src/containers/howitworks/HowItWorksSection.js]: The "How It Works" section looks pretty bad when you resize the screen to below 1020px. The image basically becomes full screen.

screen shot 2018-06-25 at 10 05 15 am

File: src/containers/jurisdictions/JurisdictionLabel.js [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    font-weight: 600;
+  }
+`;
+
+const PopulationLabel = styled.span`
+  color: ${N2};
+  display: none;
+  font-weight: normal;
+
+  @media only screen and (min-width: ${MEDIA_QUERY_JUR_MD}px) {
```
Comment (UnsungHero97) [inline src/containers/jurisdictions/JurisdictionLabel.js]: The population label disappears when the screen drops below 900px.

screen shot 2018-06-25 at 10 05 57 am

File: src/containers/howitworks/TabDisplay.js [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+            { TABS[2] }
+          </TabItem>
+          <TabItem data-name="3" onClick={this.handleTabClick} className={this.getActiveStatus(3)}>
+            { TABS[3] }
+          </TabItem>
+          <TabItem data-name="4" onClick={this.handleTabClick} className={this.getActiveStatus(4)}>
+            { TABS[4] }
+          </TabItem>
+        </Navigation>
+        { this.renderContent(this.state.tab) }
```
[the remark this answers is not in the document]
Comment (corwinc) [reply src/containers/howitworks/TabDisplay.js]: Fixed

File: src/containers/jurisdictions/JurisdictionLabel.js (hunk 1 above)
Comment (corwinc) [reply src/containers/jurisdictions/JurisdictionLabel.js]: Fixed

File: src/containers/howitworks/Content01.js (hunk 2 above)
Comment (corwinc) [reply src/containers/howitworks/Content01.js]: Fixed

File: src/containers/howitworks/Content02.js (hunk 3 above)
Comment (corwinc) [reply src/containers/howitworks/Content02.js]: Fixed

File: src/containers/howitworks/Content03.js (hunk 4 above)
Comment (corwinc) [reply src/containers/howitworks/Content03.js]: Fixed

File: src/containers/howitworks/Content04.js (hunk 5 above)
Comment (corwinc) [reply src/containers/howitworks/Content04.js]: Fixed

File: src/containers/howitworks/HowItWorksSection.js (hunk 6 above)
Comment (corwinc) [reply src/containers/howitworks/HowItWorksSection.js]: Fixed, but probably needs some design in put from Paul (both small and med browser sizes, and for everything)

File: src/containers/jurisdictions/JurisdictionLabel.js (hunk 7 above)
Comment (corwinc) [reply src/containers/jurisdictions/JurisdictionLabel.js]: Fixed

**Comment by corwinc** [comment]
Fixed everything related to your comments. Also includes:
- Merged develop and fixed related lint errors
- Fixed inaccurate line heights
- Added links as needed to How it Works section
- Renamed style.js to Style.js

**Pull request merged**